README.md

```diff
@@ -87,16 +89,23 @@ created.  The expected body is a jsonapi singular collection of
 ``` javascript
 {
   "instance_builds": {
+    "role": "worker",
     "site": "org",
     "env": "staging",
     "instance_type": "c3.2xlarge",
+    "slack_channel": "#general",
     "count": 4,
-    "queue": "docker"
+    "queue": "docker",
+    "bootInstance": true
```

Comment:
This should probably be `boot_instance` to match the other options (like `slack_channel` and `instance_type`).